entity/entity.proto

```diff
@@ -51,17 +49,20 @@ message BankPaymentMethod {
   string holder_name = 3;
   string bank_key = 4;
   string bank_account_number = 5;
-  string currency = 6;
+  string supported_currency = 6;
 }
 
 message CryptoPaymentMethod {
   bytes identifier = 1;
-  string account_address = 2;
+  string to = 2;
+  string chain_uri = 3;
+  string supported_currency = 4;
 }
 
-message PayPalMethod {
+message OtherPayment {
   bytes identifier = 1;
```

Comment:
Do we need a type here? `PayPal`, `Venmo`, `SquareCash` ...

Reply:
let's not hardcode these values in the code ;) That would be very restricting.

Reply:
Ok, so I will keep it as it is?

Reply:
not hardcoding, just a type string

Reply:
I will add a type field.